core/execution/future/liquidation.go

```diff
@@ -78,16 +78,15 @@ func (m *Market) checkNetwork(ctx context.Context, now time.Time) error {
 	}
 	// Now that the fees have been taken care of, get the current last traded price:
 	lastTraded := m.getLastTradedPrice()
+	tradeType := types.TradeTypeNetworkCloseOutGood
 	// now handle the confirmation like you would any other order/trade confirmation
-	m.handleConfirmation(ctx, conf)
+	m.handleConfirmation(ctx, conf, &tradeType)
```

Comment:
a bit nitpick: you could have handleConfirmation take TradeType and pass by default TradeTypeDefault rather than passing a pointer and nil

Reply:
It's been a while, but I decided to pass a pointer here because I found it a bit clunky to pass the unspecified trade type value to this call in all the places where this function is being called. To my eye, having the other callers pass in `nil` communicates that the argument is not applicable, whereas passing in some other value could suggest that  `handleConfirmation(ctx, conf, <for trades of this type>)` or something like that. I also feel a bit conflicted about values being passed around that, depending on the specific value result in different behaviour (ie unspecified == ignore, `TradeTypeNetworkCloseoutGood` == set the trade type accordingly, values other than those two == ??

Because of that, I just thought either nil -> ignore, not nil -> set the trade type made more sense